### DISCUSSION TITLE
Toggle view from chart → text → chart loses existing data

### DISCUSSION THREAD

**Discussion by adamwulf** [dataFude-app/dataFude#6] [Beta feedback]
A small bug report: When I toggle a tile to text mode and then back to chart mode, it doesn't display existing data for that tile, it starts the chart off empty. It'd be great to keep existing data if possible

https://user-images.githubusercontent.com/251087/205218381-1cb2edf7-ee77-4767-a5d6-6d0845f1c3cd.mov

Thanks!

**Comment by icanzilb** [comment]
Hey Adam, I love that you're having fun with dataTile, I hope you find it useful.

I agree that it'd be a nice touch to keep the last batch of values in memory so switching to chart view can feel more polished — great idea. Putting this down to my todo list.

Marin
